packages/editor-ui/src/composables/__tests__/useCanvasPanning.test.ts

```diff
@@ -20,6 +20,11 @@ describe('useCanvasPanning()', () => {
 	let element: HTMLElement;
 	let elementRef: Ref<null | HTMLElement>;
 
+	beforeAll(() => {
+		// Mocks for useDeviceSupport
+		Object.defineProperty(window, 'matchMedia', { value: vi.fn().mockImplementation(() => true) });
```

Comment:
Can we make this global in `src/__tests__/setup.ts`? Looks like it'll be needed whenever we use `useDeviceSupport`.

Reply:
Makes sense, will update